feed/views.py

```diff
@@ -1,3 +1,100 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http.response   import JsonResponse
+from django.views           import View
+
+from .models                import Feed, Comment, ImageUrl
+from user.models            import User
+from product.models         import Product
+
+class FeedEntireView(View):    
```

Comment:
```suggestion
class FeedView(View):    
```
피드 전체 뷰라는 말이 약간 어색한 것 같아서, 피드뷰 추천드립니다 :)